meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: "pytest-playwright"
+  version: "{{ environ.get('GIT_DESCRIBE_TAG') }}"
+
+source:
+  path: .
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
its 3.8 nowadays: https://github.com/microsoft/playwright-pytest/pull/177

Reply:
> its 3.8 nowadays: #177

I was wondering, I knew 3.7 is EOL, I guess since Grayskull based it on the current PyPi package which still supports 3.7. Happy to change!